scripts/rollup/forks.js

```diff
@@ -137,6 +137,19 @@ const forks = Object.freeze({
             return 'shared/forks/ReactFeatureFlags.testing.www.js';
         }
         return 'shared/forks/ReactFeatureFlags.testing.js';
+      case 'react':
+        switch (bundleType) {
+          case FB_WWW_DEV:
+          case FB_WWW_PROD:
+          case FB_WWW_PROFILING:
+            return 'shared/forks/ReactFeatureFlags.www.js';
+          case RN_FB_DEV:
+          case RN_FB_PROD:
+          case RN_FB_PROFILING:
+            return 'shared/forks/ReactFeatureFlags.native-fb.js';
+          default:
+            return 'shared/ReactFeatureFlags.js';
```

Comment:
This should just be missing or null? See the case at the bottom.

Reply:
That fails eslint `no-fallthrough`, and I think for good reason - it's nice to explicitly know which flags are used in the default case. For example, I didn't really know or believe that we used the non-scoped `ReactFeatureFlags.js` file for all of React. If this fell through (especially if there's a case added after this before the default case later), it would be more difficult to see which flag is used.

Reply:
I'm just a bit confused why this particular case is different from other switch'es above which don't have a default case.

Reply:
I see, yes this could just return.

Reply:
This was changed in https://github.com/facebook/react/pull/19522 anyway 👍